src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Builders/DataSetPublishingServiceBuilder.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+using Moq;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Tests.Builders;
```

Comment:
Have you considered splitting the builders up into different namespaces to separate model builders from service builders? There's probably pros and cons to both consolidated namespaces and separate namespaces but I was just thinking it might be easier to browsing through the solution to have them grouped if you were trying to find a specific one, and there's a clearer distinction of responsibility.